src/lsp/cobol_parser/parser_engine.ml

```diff
@@ -284,6 +285,20 @@ let post_production ({ preproc = { tokzr; _ }; _ } as ps)
       let tokzr, token, tokens
         = Tokzr.decimal_point_is_comma tokzr token tokens in
       update_tokzr ps tokzr, token, tokens
+  | Post_function_specifier spec ->
+    begin match spec with
+    | IntrinsicFunctionSpecifier intrinsics ->
+        let intrinsics = List.map (~&) intrinsics in
+        let tokzr, token, tokens
+          = Tokzr.intrinsic_functions_specifier ~intrinsics tokzr token tokens in
+        update_tokzr ps tokzr, token, tokens
+    | IntrinsicFunctionAllSpecifier ->
+        let tokzr, token, tokens
+          = Tokzr.intrinsic_functions_specifier tokzr token tokens in
+        update_tokzr ps tokzr, token, tokens
+    | _ ->
```

Comment:
Please list the missing cases here. Also, extract this to a smaller function above `register_intrisics ?intrinsics tokzr token tokens = …`, and avoid the nested pattern:
```ocaml
| Post_function_specifier IntrinsicFunctionSpecifier intrinsics ->
    …
| Post_function_specifier IntrinsicFunctionAllSpecifier ->
    …
```